mongodb_consistent_backup/Upload/Rsync/Rsync.py

```diff
@@ -56,7 +56,7 @@ def init(self):
     def rsync_info(self):
         if not self._rsync_info:
             output = check_output([self.rsync_binary, "--version"])
-            search = re.search("^rsync\s+version\s([0-9.-]+)\s+protocol\sversion\s(\d+)", output)
+            search = re.search(r"^rsync\s+version\s([0-9.-]+)\s+protocol\sversion\s(\d+)", output)
```

Comment:
@dschneller could you help explain what this change does?

Reply:
Sure. This tells python that it should _not_ interpret the escape sequences inside the string, but instead leave them as is. That's desirable for regular expressions, because otherwise you could run into conflicts when using backslashes with their special meanings for regexes.

I personally haven't even used the rsync upload, but this was pointed out by flake8 and kept me from building with the make files. That's why this change is also included in the other PRs I have sent -- each one of them would fail the Travis builds without it, so I wonder how it got in there in the first place :-)

Here's the Python documentation for raw strings: https://docs.python.org/2/reference/lexical_analysis.html#string-literals

Reply:
Thanks @dschneller, that makes more sense now. This PR looks good to me